pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -1069,13 +1069,15 @@ class Build {
                 }
 
                 // Sign and archive jobs if needed
-                try {
-                    context.timeout(time: buildTimeouts.SIGN_JOB_TIMEOUT, unit: "HOURS") {
-                        sign(versionInfo)
+                if (enableInstallers) {
+                    try {
+                        context.timeout(time: buildTimeouts.SIGN_JOB_TIMEOUT, unit: "HOURS") {
+                            sign(versionInfo)
+                        }
+                    } catch (FlowInterruptedException e) {
+                        context.println "[ERROR] Sign job timeout (${buildTimeouts.SIGN_JOB_TIMEOUT} HOURS) has been reached. Exiting..."
+                        throw new Exception()
```

Comment:
should throw Exception(e)

Reply:
There shouldn't be a need for that as the `FlowInterruptedException` is not actually the problem in the timeout situation, it's just signalling that a timeout has been hit